Retarus/trigger_retarus_events.json

```diff
@@ -0,0 +1,22 @@
+{
+    "description": "",
+    "docker_parameters": "",
+    "name": "Retarus",
+    "arguments": {
+        "$schema": "http://json-schema.org/draft-07/schema#",
+        "properties": {
+            "ws_url": {"description": "Retarus websocket URL", "type": "string", "format": "uri"},
+            "ws_key": {"description": "Retarus websocket auth key", "type": "string"},
+            "intake_key": {"description": "Intake key", "type": "string"},
+            "kafka_topic": {"description": "Kafka topic to send events to", "type": "string"}
+        },
+        "required": ["ws_url", "ws_key", "intake_key", "kafka_topic"],
+        "secrets": ["ws_key", "intake_key"],
+        "title": "Retarus events",
+        "type": "object"
+    },
+    "uuid": "",
```

Comment:
Must be provided

Reply:
How do we create it? I generate one at random and just check it's not used by any other module?

Reply:
Yes. If you didn't do it for the other trigger manifest then you should also generate one